### ISSUE TITLE
Keyboard navigation to image caption toolbar

### ISSUE THREAD

**Issue by ajlende** [WordPress/gutenberg#26114]
**Describe the bug**
Cannot navigate into the image caption toolbar with the keyboard.

**To reproduce**
1. Create image block
2. Enter some text
3. Try to change that text color with just the keyboard
4. Unable to navigate into toolbar

**Expected behavior**
Should be able to navigate into the image caption toolbar with the keyboard.

**Screenshots**
<img width="164" alt="caption toolbar" src="https://user-images.githubusercontent.com/5129775/96003616-0749d480-0e00-11eb-8301-ef8f33512d84.png">

**Editor version (please complete the following information):**
- WordPress version: 5.5.1
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? Gutenberg plugin
- If the Gutenberg plugin is installed, which version is it? Master branch (b126b3f)

**Desktop:**
 - OS: MacOS
 - Browser: Chrome
 - Version: 85.0.4183.121

**Comment by alexstine** [comment]
I can confirm this is a bug. There seems to be no way to interact with any type of toolbar from the caption field, the only toolbar accessible is the main block toolbar, no formatting options.

**Comment by diegohaz** [comment]
Just made #28420, which should hopefully fix this issue.
